src/aws/osml/image_processing/gdal_tile_factory.py

```diff
@@ -35,6 +35,7 @@ def __init__(
         tile_compression: GDALCompressionOptions = GDALCompressionOptions.NONE,
         output_type: Optional[int] = None,
         range_adjustment: RangeAdjustmentType = RangeAdjustmentType.NONE,
+        logger: Optional[logging.Logger] = default_logger,
```

Comment:
Please don't do this. Passing loggers around this way is bad practice especially in a low level library like the imagery toolkit.

Reply:
fixed in latest version.